lib/sentry_logging.rb

```diff
@@ -31,13 +29,18 @@ def log_exception_to_sentry(exception, extra_context = {}, tags_context = {}, le
     end
     rails_logger(level, exception.backtrace.join("\n")) unless exception.backtrace.nil?
   end
-  # rubocop:enable Metrics/CyclomaticComplexity
-  # rubocop:enable Metrics/PerceivedComplexity
 
-  def normalize_level(level)
+  def normalize_level(level, exception = nil)
```

Comment:
looks like `normalize_level` is only used in two place.  Maybe just explicitly pass in `nil` on line 7?  Or you could even pass in a symbol like `:no_exception` (maybe assigned to a constant?) and it would fall through the same as `nil`.  The default argument doesn't add a lot in my opinion and the scope of the use is pretty narrow.  I would say it would necessitate extra test cases, but it looks like `lib/sentry_logging.rb` is untested.